server/migrations/20210925190654-create-user.ts

```diff
@@ -1,43 +1,43 @@
-import { UUIDV4 } from "sequelize";
+import { Sequelize, QueryInterface, DataTypes, UUIDV4 } from "sequelize";
 
 module.exports = {
-  up: async (queryInterface: any, Sequelize: any) => {
+  up: async (queryInterface: QueryInterface, Sequelize: Sequelize) => {
```

Comment:
Since you defined Userid field in the model, you should add it here too

Reply:
Do you mean in create-post.ts, not create-user.ts?

Reply:
yes, my bad